### PR TITLE
Handle OCIO environment variable

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-2519

PURPOSE: OCIO environment variable can break the feature in some cases. Change requested by RE team.

### CODE REVIEW

File: FireRender.Maya.Src/Context/TahoeContext.cpp [hunk 1]
```diff
@@ -252,6 +253,18 @@ void TahoeContext::setupContext(const FireRenderGlobalsData& fireRenderGlobalsDa
 
 	// OCIO
 	{
+		const auto& eVars = EnvironmentVarsWrapper<char>::GetEnvVarsTable();
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/TahoeContext.cpp:256]: could we get rid of "auto" ?

**Review by Vsevolod1983 — CHANGES_REQUESTED** [review]

File: FireRender.Maya.Src/Context/TahoeContext.cpp [hunk 2]
```diff
@@ -252,6 +253,18 @@ void TahoeContext::setupContext(const FireRenderGlobalsData& fireRenderGlobalsDa
...
+		auto envOCIOPath = eVars.find("OCIO");
```
Comment (Vsevolod1983) [inline FireRender.Maya.Src/Context/TahoeContext.cpp:257]: could we get rid of "auto" ?

Comment (DMotorygin) [reply FireRender.Maya.Src/Context/TahoeContext.cpp:257]: envOCIOPath is ::const_iterator
Haven't we decided to use auto for iterators?

File: FireRender.Maya.Src/Context/TahoeContext.cpp (hunk 1 above)
Comment (DMotorygin) [reply FireRender.Maya.Src/Context/TahoeContext.cpp:256]: done

**Review by Vsevolod1983 — APPROVED** [review]

**Pull request merged**